tests/test_version_range.py

```diff
@@ -393,3 +393,20 @@ def test_npm_advisory_version_range_parse(test_case):
         string=test_case["npm_native"],
     )
     assert str(result) == test_case["expected_vers"]
+
+
+def test_inverse():
```

Comment:
You need tests for the lower level constraint.invert e.g.,  for https://github.com/nexB/univers/pull/92/files#diff-c77d34029ce1991650937489359eef45029f37be4f1f27c21f56e50e64209d9aR138 too